set.mm

```diff
@@ -281338,12 +281338,15 @@ we show (in ~ tgcl ) that ` ( topGen `` B ) ` is indeed a topology (on
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
-  $( Reverse closure for the subspace topology.  (Contributed by Mario
-     Carneiro, 19-Mar-2015.)  (Revised by Mario Carneiro, 1-May-2015.) $)
-  restrcl $p |- ( ( J |`t A ) e. Top -> ( J e. _V /\ A e. _V ) ) $=
-    ( crest co ctop wcel c0 wceq cvv wa wn 0opn n0i syl cxp wfn cdm restfn fndm
-    ax-mp ndmov nsyl2 ) BACDZEFZUCGHZBIFAIFJUDGUCFUEKUCLUCGMNBAICCIIOZPCQUFHRUF
-    CSTUAUB $.
+  ${
+    $d j x y $.
```

Comment:
The original theorem had no distinct variable condition, so it should be kept (as OLD theorem at least).

Reply:
Doesn't much matter to me. I've dropped the change to set.mm here.